release-notes/docs/3.13.0.md

```diff
@@ -0,0 +1,85 @@
+# 3.13.0 Release Notes
+
+- [Known issues](#known-issues)
+- [Upgrade notes](#upgrade-notes)
+- [Breaking changes](#breaking-changes)
+- [UI/UX changes](#uiux-changes)
+- [Highlights](#highlights)
+- [And more...](#and-more)
+
+## Known issues
+
+None.
+
+## Upgrade notes
+
+### Breaking changes
+
+None.
+
+### UI/UX changes
+
+- [#7235](https://github.com/medic/cht-core/issues/7235): Make it possible to display previous task due date formatting via configuration
+
+
+## Highlights
+
+### Collect form data from external Android apps
+
+The CHT now supports forms that collect data from generic 3rd-party Android apps. Users can launch the apps from the form and then the output from the app is captured by the CHT and stored with the report. See the [CHT documentation](https://docs.communityhealthtoolkit.org/apps/reference/forms/app/#android-app-launcher) for more information on how to use this feature.
+
+> An example use case for this functionality is [the CHT reference application](https://docs.communityhealthtoolkit.org/apps/examples/covid-rdt-reference-app/) for COVID-19 point-of-care testing with Rapid Diagnostic Tests (RDT). This app provisions and captures the RDT data by integrating with Dimagi’s [RD-Toolkit](https://github.com/dimagi/rd-toolkit/) app that guides health workers through the use of the RDT.
+
+This feature requires the CHT External App Launcher functionality included in [v0.10.0](https://github.com/medic/cht-android/releases/tag/v0.10.0) of cht-android. 
+
+- [#6981](https://github.com/medic/cht-core/issues/6981): Support collecting form data from external Android apps
+
+### Household Tasks Landing Page
+
+When the user has completed a task for a particular household and there are additional uncompleted task for that household, a landing page is shown with a list of the remaining tasks for that household. This allows the user to easily access all the tasks remaining for their current visit. 
+
+- [#5886](https://github.com/medic/cht-core/issues/5886): Make it easier to complete all tasks at a household in the same visit
```

Comment:
It's also worth noting that this is only shown when a new permission is added to a user's role so the default behaviour is the same as previous releases.

Reply:
Also this is another great place to show a screenshot of the improvement!